backend/build.gradle.kts

```
@@ -69,3 +81,180 @@ dependencies {
+            @Suppress("UNCHECKED_CAST")
+            val applicationJson = content["application/json"] as? MutableMap<String, Any?>
+            if (applicationJson != null) {
+                @Suppress("UNCHECKED_CAST")
+                val examples = applicationJson["examples"] as? MutableMap<String, Any?>
+                @Suppress("UNCHECKED_CAST")
+                examples?.values?.filterIsInstance<MutableMap<String, Any?>>()?.forEach { exampleObj ->
+                    val valueStr = exampleObj["value"] as? String
+                    if (valueStr != null && (valueStr.trim().startsWith("{") || valueStr.trim().startsWith("["))) {
+                        try {
+                            exampleObj["value"] = jsonSlurper.parseText(valueStr)
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)